agent/config.go

```diff
@@ -17,7 +17,7 @@ const (
 
 	// TCPSocketReadDeadline specifies the maximum time the TCP socket will wait
 	// to receive data.
-	TCPSocketReadDeadline = 500 * time.Millisecond
+	TCPSocketReadDeadline = 5000 * time.Millisecond
```

Comment:
Curious to know what's the rationale being this 10 fold increase?
Not objecting to the change at all, just curious!

Reply:
Well, 500 ms seemed a bit tight